source/stylesheets/about.css.scss

```diff
@@ -86,6 +86,16 @@ body.index, body.about {
   @media screen and (min-width: 2000px) { // Stretch background width when wider than source image
```

Comment:
we should extract these widths into variables. is there a way to have the whole media query blocks as reusable bits with Sass?